pkg/generic/binarythrift_codec.go

```diff
@@ -51,22 +50,16 @@ func (c *binaryThriftCodec) Marshal(ctx context.Context, msg remote.Message, out
 	var transBuff []byte
 	var ok bool
 	if msg.RPCRole() == remote.Server {
+		// Business error only works properly when using TTHeader and HTTP2 transmission protocols
+		// If there is a business error, data.(*Result).Success will be nil, and an empty payload will be constructed here to return
+		if msg.RPCInfo().Invocation().BizStatusErr() != nil {
+			wb := thrift.NewWriteBinary()
```

Comment:
No need to create wb every time, use a global variable to reduce object mallocing